week6/src/Task.js

```diff
@@ -1,20 +1,32 @@
 import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router'
 import './App.css';
-
+import {Link} from "react-router-dom"
+import {useRouteError} from "react-router-dom"
 
 export default function Task() 
 {
-  const { id } = useParams();
+  const error = useRouteError();
+  const { value } = useParams();
   const storedTask = JSON.parse(localStorage.getItem('items'));
   return Object.keys(storedTask).map((obj, i)=>{
-    if(storedTask[obj].id ==id){
+    if(storedTask[obj].value ==value){
       return (
         <div className='taskpage'>
           <h1>{storedTask[obj].value}</h1>
           <hr></hr>
         </div>
        )
     }
+    else{
+      return(
+        <div>
+           <h1>404</h1>
+        <h2>PAGE NOT FOUND</h2>
+        <p>it appears the task you are looking for does not exist</p>
+        <p>Click link to go back to <Link to='/'>HomePage</Link></p>
+        </div>
+       );
+    }
   })
```

Comment:
This doesn't work as expected. See the screenshot below

<img width="443" alt="Screenshot 2023-07-31 at 08 49 13" src="https://github.com/thelightreaper123/internship/assets/29612423/12029abc-e207-4332-ab9e-f6b4ef58d858">